airflow/www/app.py

```diff
@@ -61,8 +61,10 @@ def create_app(config=None, testing=False):
 
         admin = Admin(
             app, name='Airflow',
-            static_url_path='/admin',
-            index_view=views.HomeView(endpoint='', url='/admin', name="DAGs"),
+            static_url_path=conf.get_url_prefix() + '/admin',
+            index_view=views.HomeView(endpoint='',
+                                      url=conf.get_url_prefix() + '/admin',
+                                      name="DAGs"),
```

Comment:
I'm not sure what it would break, but I feel that we should pass a `url=conf.get_url_prefix()` call to the Admin() constructor too. This might break everything, but this would map down to `app.register_blueprint(admin, url_prefix=admin.url)` under the hood which feels like the right thing to do.

Reply:
Hopefully including this would mean that we don't have to pass url_prefix to all of the render calls though.

Reply:
I just test it it does not change anything :(